src/server/src/main/java/io/cassandrareaper/ReaperApplication.java

```diff
@@ -71,14 +71,14 @@
 import io.dropwizard.client.HttpClientBuilder;
```

Comment:
The constructor `public ReaperApplication(AppContext context)` doesn't  seem used anymore, maybe it should be deleted?

Reply:
Good catch! It was probably used in the integration tests through reflection. I'll remove it.